src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/CopyRatioSegmenter.java

```diff
@@ -24,27 +25,23 @@
      *                          include in the model.  Hidden states are pruned as the model is learned.
      */
     public CopyRatioSegmenter(final int initialNumStates, final List<SimpleInterval> positions, final List<Double> data) {
-        super(initialNumStates, positions, data);
+        super(initialNumStates, positions, data, initialLog2CopyRatios(initialNumStates));
+        logCoverageCauchyWidth = DEFAULT_INITIAL_CAUCHY_WIDTH;
     }
 
     /**
      * evenly-spaced log-2 copy ratios
      * @param K the initial number of hidden states
      */
-    @Override
-    protected void initializeHiddenStateValues(final int K) {
-        hiddenStateValues = GATKProtectedMathUtils.createEvenlySpacedPoints(-3, 2, K);
-        hiddenStateValues[NEUTRAL_VALUE_INDEX] = NEUTRAL_LOG_2_COPY_RATIO;
+    private static List<Double> initialLog2CopyRatios(final int K) {
+        final List<Double> result = Doubles.asList(GATKProtectedMathUtils.createEvenlySpacedPoints(-3, 2, K));
+        result.set(NEUTRAL_VALUE_INDEX, NEUTRAL_LOG_2_COPY_RATIO);
```

Comment:
Right, just to clarify my original comment: The javadoc for this method specifies that it will return "evenly-spaced log-2 copy ratios," but that is not necessarily what is produced.  Rather, it produces a list with log-2 copy ratios that would be evenly spaced were it not for the element at NEUTRAL_VALUE_INDEX being set to NEUTRAL_LOG_2_COPY_RATIO.

I also think it would be more natural to generate `K - 1` states and then use `result.add(NEUTRAL_VALUE_INDEX,  NEUTRAL_LOG_2_COPY_RATIO)`, but this is perhaps a matter of taste.